homepage-Hershey.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
The css code needs to be linked in order to show the style of the page. You can do that just by adding
`<link rel="stylesheet" href="homepage-Hershey.css">` in line 6.